thoth/storages/graph/postgres.py

```diff
@@ -3475,6 +3475,15 @@ def update_kebechet_github_installations_on_is_active(self, slug: str) -> bool:
                 return True
             return False
 
+    def get_kebechet_github_installations_active(self) -> List[str]:
+        """Return all of the repositories wiht the active Kebechet installation."""
```

Comment:
`Get all active repositories names with Kebechet installation.`

Reply:
done, i've also added `active` to the docstring, is that alright?